lib/MetaCPAN/Script/Release.pm

```diff
@@ -490,6 +498,73 @@ sub import_tarball {
     }
 }
 
+sub find_main_module {
+    my $self = shift;
+    my ( $mod, $distribution ) = @_;
+
+    my @modules = @{$mod};
+
+    my $min_level          = 100;
+    my @distribution_parts = split /-/, $distribution;
+    my $distribution_name  = join '', @distribution_parts;
+
+    if ( scalar @modules eq 1 ) {
+
+        # there is only one module and it will become the main_module
+        my $main_module = shift @modules;
+        $self->main_module( $main_module->module );
+        return;
+    }
+    else {
```

Comment:
Since you're returning in the `if` above, the `else` is not necessary.  If you get rid of it, you can save yourself one level of indentation.